tests/console/zypper_lifecycle.pm

```diff
@@ -42,22 +42,16 @@ our $date_re = qr/[0-9]{4}-[0-9]{2}-[0-9]{2}/;
 sub run {
     diag('fate#320597: Introduce \'zypper lifecycle\' to provide information about life cycle of individual products and packages');
 
-    # Workaround for bsc#1166549, zypper lifecycle error with 'Error building the cache'.
-    # Currently the only workaround option is to run zypper-lifecycle as root.
-
-    # Now we just hit bsc#1166549 on s390x and ppc64le, so we just need root console
-    # on these two platforms.
-    my $needs_root_console = 0;
-    if ((is_s390x || is_ppc64le) && is_sle('>=15-SP3') && is_upgrade) {
-        $needs_root_console = 1;
-        record_soft_failure 'bsc#1166549 - zypper lifecycle failed to create metadata cache directory';
-    }
     select_console 'root-console';
-
+    # first we'd make sure that zypper cache dir have 0755 permission
```

Comment:
Huajian,

Any reason found by your team that the default permission 0755 get changed? umask is changed, right?

Reply:
No.

Reply:
Got it, then can you please add some more comments here, e.g. bug with workaround?

Reply:
OK